pkg/client/unversioned/extensions.go

```diff
@@ -140,11 +140,7 @@ func setExtensionsDefaults(config *Config) error {
 	}
 	// TODO: Unconditionally set the config.Version, until we fix the config.
 	//if config.Version == "" {
-	gv, err := unversioned.ParseGroupVersion(g.GroupVersion)
-	if err != nil {
-		return err
-	}
-	config.GroupVersion = &gv
+	config.GroupVersion = &g.GroupVersion
```

Comment:
We probably need a copy to make sure we don't leak references to the actual `GroupVersion`, right?

```go
copyOfGV := g.GroupVersion
config.GroupVersion = &gv
```

?